docs/UserGuide.md

```diff
@@ -72,9 +75,40 @@ QuickCache is a **desktop app for managing flashcards, optimized for use via a C
 
 Shows a message explaning how to access the help page.
 
+Format: `help`
+
 ![help message](images/helpMessage.png)
 
-Format: `help`
+### Creating a new flashcard : You can add a new flashcard to the list.
```

Comment:
The preface of the feature should not in the header. Refer to my implementation of the delete section